components/demos/Accordion/css/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ChevronDownIcon } from '@radix-ui/react-icons';
+import * as Accordion from '@radix-ui/react-accordion';
+import './styles.css';
```

Comment:
nit, let's try and keep a consistent order for those:

```suggestion
import React from 'react';
import * as Accordion from '@radix-ui/react-accordion';
import classNames from 'classnames';
import { ChevronDownIcon } from '@radix-ui/react-icons';
import './styles.css';
```